setup.py

```diff
@@ -7,7 +7,7 @@
 REQUIREMENTS = [
     'django-cms>=3.7',
     'django-filer>=1.7',
-    'django-treebeard>=4.3,<4.5',
+    'django-treebeard>=4.3',
```

Comment:
```suggestion
```
Let's remove this line altogether. djangocms-googlemaps does not directly depend on django-treebeard.

Reply:
@fsbraun , I understand that djangocms-googlemap can't be used without djangocms, but still we have a reference in that library to `django-treebeard` here - https://github.com/django-cms/djangocms-googlemap/blob/42a5b81dcfd69a3559434e23772c6c8fcda0cae9/djangocms_googlemap/migrations/0005_create_nested_plugins.py#L7C1-L7C1.

So should I remove it or leave it?